### ISSUE TITLE
using macros for mitochondrial net work analysis, i have dowloaded image J fiji version https://fiji.sc/.  i a m not sure about the  macro tried to follow tha paper simple ImageJ macro tool for analyzing mitochondrial network morphology in mammalian cell culture. i am getting up to skeltonization but my results window is not showing as it is in the paper. is something i have o do withe macro? do i need to dowload specific macros?

### ISSUE THREAD

**Issue by saritarag** [ScienceToolkit/MiNA#4]

**Comment by ScienceToolkit** [comment]
Hello,

The paper and associated macro behaviour is regarding the version 1 macros, which are still available in the version_1 folder. Version 2 macros behave differently and are being tinkered on actively at the moment. 

I am going to take a guess and say the macro you installed is one of the version 2 files. They behave differently and I haven't gotten around to making the documentation for using them yet or thoroughly testing them.

_In short, installing the MiNA_V110.ijm file will likely provide the behaviour you are expecting._

Cheers,
Andrew

**Comment by saritarag** [comment]
Thanks Andrew.
